pygenometracks/tracksClass.py

```diff
@@ -494,6 +495,30 @@ def parse_tracks(self, tracks_file):
             # The track_options will be checked for the file paths:
             track_options = self.check_file_exists(track_options,
                                                    tracks_file_path)
+
+            # If a big bedgraph is used a warning is printed:
+            if track_class == BedGraphTrack:
+                file_size = os.path.getsize(track_options['file'])
```

Comment:
This could be a function with max_file_size and maybe a more general error message?

Reply:
I think it should not be an error but a warning. What I like with the 2 different messages here it that they give a solution (bedgraph -> bigwig, gtf -> bed). A more general message would loose these advise. But I agree, this could be in the README and we could use a general message to refer to it. I will do that.

Reply:
Na then keep it. I just thought that a function could reuse some code. maybe that is overdoing it ... leave it like it is if you prefer :)